src/marionette.rs

```diff
@@ -1357,6 +1360,14 @@ impl ToMarionette for JavascriptCommandParameters {
     }
 }
 
+impl ToMarionette for ActionsParameters {
+    fn to_marionette(&self) -> WebDriverResult<BTreeMap<String, Json>> {
+        Ok(try_opt!(self.to_json().as_object(),
+                    ErrorStatus::UnknownError,
```

Comment:
Not really; anything other than an object here would be a bug in webdriver-rust rather than invalid input. It "can't" happen, but the type system can't prove it because enum variants aren't types, so you can't specify that the `to_json()` must return a `Json::Object` (and in other cases it need not be an object).